tools/link.py

```diff
@@ -786,10 +791,11 @@ def phase_linker_setup(options, linker_args):  # noqa: C901, PLR0912, PLR0915
 
   if settings.WASM_ESM_INTEGRATION:
     diagnostics.warning('experimental', '-sWASM_ESM_INTEGRATION is still experimental and not yet supported in browsers')
-    default_setting('EXPORT_ES6', 1)
     default_setting('MODULARIZE', 'instance')
-    if not settings.EXPORT_ES6 or settings.MODULARIZE != 'instance':
-      exit_with_error('WASM_ESM_INTEGRATION requires EXPORT_ES6 and MODULARIZE=instance')
+    if options.oformat != OFormat.MJS:
+      exit_with_error('WASM_ESM_INTEGRATION is only compatible with .mjs output files')
```

Comment:
Is this a new restriction?

Reply:
Its not new I'm just trying to be consistent with the same error message for `-sMODULARIZE=instance` below.  I'm rephrased both of them.